contrib/index.html

```diff
@@ -20,206 +20,34 @@ <h1 class="welcome-message-title">
-        <div class="code">
-          <code>@include outer-container;</code>
+      <main role="main" class="examples">
+        <div class="column--full">
+          <h3>Basic</h3>
         </div>
-        <section class="page-section fifth">
-          <div class="box">
-<pre>
-  .box {
-    @include span-columns(3);
-    @include omega(4n);
-  }
-</pre>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-            <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column"></div>
+        <div class="box column--thirds">
+          <div class="box--nested column--thirds--alt">
+          </div>
+          <div class="box--nested column--thirds--alt">
+          </div>
+          <div class="box--nested column--thirds--alt">
           </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-          <div class="fifth-box page-section-inner">
-             <code>.box</code>
-          </div>
-        </section>
-        <h3>Media Queries</h3>
-        <section class="page-section sixth">
-<pre>
-  @include outer-container;
-  $mobile: new-breakpoint(max-width 500px 4);
-</pre>
-          <div class="page-section-inner sixth-alpha">
-<pre>
-  @include span-columns(4);
-
-  @include media($mobile) {
-    @include span-columns(3);
-  }
-</pre>
-          </div>
-          <div class="page-section-inner sixth-beta">
-<pre>
-  @include span-columns(8);
-  @include media($mobile) {
-    @include span-columns(3);
-  }
-</pre>
-          </div>
-        </section>
+        </div>
+        <div class="box column--thirds">
+        </div>
+        <div class="box column--thirds">
+        </div>
       </main>
     </body>
   </html>
```

Comment:
Ah okay. Your comment was showing up on the deleted side so I thought you were just curious. Yep, good catch on that spacing.